test/checkout.int.t.sol

```diff
@@ -991,12 +992,13 @@ contract Checkout_buyingWithCustomFee is Checkout {
       owner,
       owner,
       certificateAmount,
-      customFee
+      customFee,
+      customPriceMultiple
     );
     vm.stopPrank();
 
     Vm.Log[] memory entries = vm.getRecordedLogs();
-    uint256 createCertificateEventIndex = 6;
+    uint256 createCertificateEventIndex = 7;
```

Comment:
now there's also a "SetPriceMultiple" event (or whatever it's actually called) happening before the creation of the certificate